core/src/banking_stage.rs

```diff
@@ -75,11 +75,11 @@ pub const FORWARD_TRANSACTIONS_TO_LEADER_AT_SLOT_OFFSET: u64 = 2;
 pub const HOLD_TRANSACTIONS_SLOT_OFFSET: u64 = 20;
 
 // Fixed thread size seems to be fastest on GCP setup
-pub const NUM_THREADS: u32 = 4;
+pub const NUM_THREADS: u32 = 6;
 
-const TOTAL_BUFFERED_PACKETS: usize = 500_000;
+const TOTAL_BUFFERED_PACKETS: usize = 700_000;
```

Comment:
Just to be clear this means:

1. Each non-vote banking thread can now hold `700,000 / (4 * 64) = 2734` `PacketBatches`, which is a 3x increase from the current `500,000/(2 * 128) = 976` PacketBatches.
2. In the worst case, this means we will use `2734 PacketBatches * 4 threads * 128 packets/PacketBatch * 1024 bytes = `1.4 GB` for buffering packets.

Reply:
yea 3x increase in number of batches across 4 threads now, but should be only 40% increase in total memory bytes from the 500k -> 700k change.